README.md

```diff
@@ -6,46 +6,192 @@
 
 ![](src/main/resources/META-INF/pluginIcon.svg)
 
+Git Machete plugin is a robust tool that simplify your git related workflow.
+It's a great completion of IntelliJ version control system.
+You can with ease rebase branches, do pull, push or reset.
+Just define what you would like your branch tree to look like and leave the rest to Git Machete.
+It gives you simple view of what's going on in your repository.
+Just look at the edge colors or status descriptions to see what should be rebased, pushed, pulled or reset.
+
+With this plugin you can simply maintain small pull request that are easy to review.
+
 This is a port of [git-machete](https://github.com/VirtusLab/git-machete#git-machete) into an IntelliJ plugin.
```

Comment:
Maybe move to after the sample workflow gif, as `Also see...`

Reply:
Also, only one of this line and line 39 (`Git Machete IntelliJ Plugin is a port of...`) is really needed